### PR TITLE
Set up react app project and deploy it to the Google Cloud App engine

### LINKED ISSUE #5: [Infra] Set up React framework (P0)

### PR DESCRIPTION
**Pull request by essencec**

### CODE REVIEW

File: capstone-project/src/logo.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (ajwang0) [inline capstone-project/src/logo.svg]: Can you remove this file

File: capstone-project/src/App.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
```
Comment (ajwang0) [inline capstone-project/src/App.js]: I know this was autogenerated from the create project CLI, but can you remove the reference to logo

File: capstone-project/src/App.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
```
Comment (ajwang0) [inline capstone-project/src/App.js]: Can you also remove this line

**Review by ajwang0 — APPROVED** [review]

**Review by ajwang0 — DISMISSED** [review]
LGTM. Nice job getting a skeleton set up

File: capstone-project/src/logo.svg (hunk 1 above)
Comment (essencec) [reply capstone-project/src/logo.svg]: Done.

**Review by ajwang0 — APPROVED** [review]

**Comment by essencec** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogleinterns%2Fplay-web-step-2020%2Fpull%2F18) for more info**.

@googlebot I signed it!

**Pull request merged**